benchmark/benchmarks.jl

```diff
@@ -85,8 +85,8 @@ SUITE["graph"]["add_union"]["agent"] =
     @benchmarkable add_agent!($graph_agent, $graph_union_model) samples = 100
 SUITE["graph"]["add_union"]["agent_pos"] =
     @benchmarkable add_agent_pos!($graph_agent, $graph_union_model) samples = 100
-SUITE["graph"]["add_union"]["agent_single"] =
-    @benchmarkable add_agent_single!($graph_agent, $graph_union_model) samples = 100
+# SUITE["graph"]["add_union"]["agent_single"] =
+#     @benchmarkable add_agent_single!($graph_agent, $graph_union_model) samples = 100
```

Comment:
Why is this commented?

Reply:
It wasn't working, I haven't looked into it yet